docs/settings/telemetry-settings.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[telemetry-settings-kbn]]
+=== Telemetry settings in Kibana
+++++
+<titleabbrev>Telemetry settings</titleabbrev>
+++++
+
+By default, the Usage Collection (also known as Telemetry) is enabled. This
+help us to learn about the {kib} features our users are more interested in, so we
+can focus our efforts on making them even better.
+
+You can control whether this data is sent from the {kib} servers or, if it should be sent 
+from the user's browser, in case there is a firewall blocking those connections from the server. Additionally, you can decide to completely disable this feature either in the config file or the {kib} UI (via Management > Kibana > Advanced Settings > Usage Data).
```

Comment:
```suggestion
from the user's browser, in case a firewall is blocking the connections from the server. Additionally, you can decide to completely disable this feature either in the config file or in {kib} via *Management > Kibana > Advanced Settings > Usage Data*.
```